app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
```

Comment:
The `show` action is required in the milestone. Kindly add it.

The `show` action is what maps to the show view and the instance variable defined here would be valuable in later projects.